modules/@apostrophecms/doc-type/ui/apos/components/AposDocContextMenu.vue

```diff
@@ -164,7 +164,7 @@ export default {
       return menu;
     },
     moduleName() {
-      if (this.context.slug.startsWith('/')) {
+      if (apos.modules[this.context.type].action === apos.modules['@apostrophecms/page'].action) {
```

Comment:
The other fix might make this unnecessary, but it's also a bit of the tail wagging the dog as-is.